packages/react-dom/src/events/accumulateTwoPhaseListeners.js

```diff
@@ -67,9 +68,11 @@ export default function accumulateTwoPhaseListeners(
               if (capture === true) {
                 dispatchListeners.unshift(callback);
                 dispatchInstances.unshift(node);
+                dispatchContainers.unshift(instance);
```

Comment:
The naming is really tripping me. We're pushing a "node" to "instances" but an "instance" to "containers". What is the mental model for what's actually in there? Can we rename these fields and variable to match, and clarify which are DOM nodes and which are Fibers and which are something else?

Reply:
I don't mean to be pedantic. I find it very difficult to wrap my mind around this, especially because we have several cases. Right now I don't even have the confidence any particular field always holds things of the same type.

Reply:
Valid point, I'll clear up the names.

Reply:
How about this:

```
_dispatchListeners
_dispatchFibers
_dispatchCurrentTargets
```

?

Reply:
I don't want to change from `dispatchInstances` as that's what the existing legacy system uses and changing that is far too risky. I can change the `currentContainers` to `_dispatchCurrentTargets` though.